eng/jacoco-test-coverage/pom.xml

```diff
@@ -187,6 +187,11 @@
       <artifactId>azure-sdk-template</artifactId>
       <version>1.0.4-beta.13</version> <!-- {x-version-update;com.azure:azure-sdk-template;current} -->
     </dependency>
+    <dependency>
+      <groupId>com.microsoft.azure</groupId>
+      <artifactId>azure-spring-boot</artifactId>
+      <version>2.2.5-beta.1</version> <!-- {x-version-update;com.microsoft.azure:azure-spring-boot;current} -->
+    </dependency>
```

Comment:
There should probably be an entry here for each spring library being added to version_client.txt below.